packages/experimental/src/components/Notifications/Notifications.js

```diff
@@ -0,0 +1,293 @@
+          </Button>
+        )}
+      </div>
+    );
+  };
+
+  const renderNotification = (group, notification, index) => {
+    return (
+      <div
+        key={`${notification.timestamp}-${notification.title}-${index}`}
+        className={[
+          `${prefix}-notifications-panel-notification`,
+          `${prefix}-notifications-panel-notification-${group}`,
+        ].join(' ')}>
+        {notification.type === 'error' && (
+          <ErrorFilled16
+            className={[
+              `${prefix}-notifications-panel-notification-status-icon`,
+              `${prefix}-notifications-panel-notification-status-icon-error`,
+            ].join(' ')}
+          />
+        )}
+        {notification.type === 'success' && (
+          <CheckmarkFilled16
+            className={[
+              `${prefix}-notifications-panel-notification-status-icon`,
+              `${prefix}-notifications-panel-notification-status-icon-success`,
+            ].join(' ')}
+          />
+        )}
+        {notification.type === 'warning' && (
+          <WarningAltFilled16
+            className={[
+              `${prefix}-notifications-panel-notification-status-icon`,
+              `${prefix}-notifications-panel-notification-status-icon-warning`,
+            ].join(' ')}
+          />
+        )}
+        {notification.type === 'informational' && (
+          <InformationSquareFilled16
+            className={[
+              `${prefix}-notifications-panel-notification-status-icon`,
+              `${prefix}-notifications-panel-notification-status-icon-informational`,
+            ].join(' ')}
+          />
+        )}
+        <div className={`${prefix}-notifications-panel-notification-content`}>
+          <p
+            className={`${prefix}-notifications-panel-notification-time-label`}>
+            {timeAgo(notification.timestamp)}
+          </p>
+          <h6 className={`${prefix}-notifications-panel-notification-title`}>
+            {notification.title}
+          </h6>
+          {notification.description &&
+            notification.description.length &&
+            renderDescription(notification.id)}
+          {notification.link &&
+            notification.link.text &&
+            notification.link.url && (
+              <Link
+                href={notification.link.url}
+                className={`${prefix}-notifications-panel-notifications-link`}>
+                {notification.link.text}
+              </Link>
+            )}
+        </div>
+      </div>
+    );
+  };
+
+  return (
+    shouldRender && (
+      <div
+        className={[`${prefix}-notifications-panel-container`].join(' ')}
+        style={{ animation: `${open ? 'fadeIn 250ms' : 'fadeOut 250ms'}` }}
+        onAnimationEnd={onAnimationEnd}>
+        <div className={`${prefix}-notifications-header-container`}>
+          <div className={`${prefix}-notifications-header-flex`}>
+            <h6 className={`${prefix}-notifications-header`}>Notifications</h6>
```

Comment:
A h6?

Reply:
I used this just to get the right type size, but I can change it to an `h1` and use the carbon mixin to get the correct type size that way.